### ISSUE TITLE
[test failure: 5.0] LogStash::Agent metrics after config reloading when reloading a bad config sets the failure reload timestamp

### ISSUE THREAD

**Issue by colinsurprenant** [elastic/logstash#7212]
https://logstash-ci.elastic.co/job/elastic+logstash+5.0+multijob-os-compatibility/os=amazon/275/console

```
Failures:

  1) LogStash::Agent metrics after config reloading when reloading a bad config sets the failure reload timestamp
     Failure/Error: expect(value).to be_a(LogStash::Timestamp)
       expected nil to be a kind of LogStash::Timestamp
     # /var/lib/jenkins/workspace/elastic+logstash+5.0+multijob-os-compatibility/os/amazon/logstash-core/spec/logstash/agent_spec.rb:491:in `(root)'
     # /var/lib/jenkins/workspace/elastic+logstash+5.0+multijob-os-compatibility/os/amazon/vendor/bundle/jruby/1.9/gems/rspec-wait-0.0.9/lib/rspec/wait.rb:46:in `(root)'
     # /var/lib/jenkins/workspace/elastic+logstash+5.0+multijob-os-compatibility/os/amazon/rakelib/test.rake:42:in `(root)'
     # /var/lib/jenkins/workspace/elastic+logstash+5.0+multijob-os-compatibility/os/amazon/vendor/bundle/jruby/1.9/gems/rake-12.0.0/lib/rake/task.rb:250:in `execute'
     # /var/lib/jenkins/workspace/elastic+logstash+5.0+multijob-os-compatibility/os/amazon/vendor/bundle/jruby/1.9/gems/rake-12.0.0/lib/rake/task.rb:250:in `execute'
     # /var/lib/jenkins/workspace/elastic+logstash+5.0+multijob-os-compatibility/os/amazon/vendor/bundle/jruby/1.9/gems/rake-12.0.0/lib/rake/task.rb:194:in `invoke_with_call_chain'
     # /var/lib/jenkins/workspace/elastic+logstash+5.0+multijob-os-compatibility/os/amazon/vendor/bundle/jruby/1.9/gems/rake-12.0.0/lib/rake/task.rb:187:in `invoke_with_call_chain'
     # /var/lib/jenkins/workspace/elastic+logstash+5.0+multijob-os-compatibility/os/amazon/vendor/bundle/jruby/1.9/gems/rake-12.0.0/lib/rake/task.rb:180:in `invoke'
     # /var/lib/jenkins/workspace/elastic+logstash+5.0+multijob-os-compatibility/os/amazon/vendor/bundle/jruby/1.9/gems/rake-12.0.0/lib/rake/application.rb:152:in `invoke_task'
     # /var/lib/jenkins/workspace/elastic+logstash+5.0+multijob-os-compatibility/os/amazon/vendor/bundle/jruby/1.9/gems/rake-12.0.0/lib/rake/application.rb:108:in `top_level'
     # /var/lib/jenkins/workspace/elastic+logstash+5.0+multijob-os-compatibility/os/amazon/vendor/bundle/jruby/1.9/gems/rake-12.0.0/lib/rake/application.rb:108:in `top_level'
     # /var/lib/jenkins/workspace/elastic+logstash+5.0+multijob-os-compatibility/os/amazon/vendor/bundle/jruby/1.9/gems/rake-12.0.0/lib/rake/application.rb:117:in `run_with_threads'
     # /var/lib/jenkins/workspace/elastic+logstash+5.0+multijob-os-compatibility/os/amazon/vendor/bundle/jruby/1.9/gems/rake-12.0.0/lib/rake/application.rb:102:in `top_level'
     # /var/lib/jenkins/workspace/elastic+logstash+5.0+multijob-os-compatibility/os/amazon/vendor/bundle/jruby/1.9/gems/rake-12.0.0/lib/rake/application.rb:80:in `run'
     # /var/lib/jenkins/workspace/elastic+logstash+5.0+multijob-os-compatibility/os/amazon/vendor/bundle/jruby/1.9/gems/rake-12.0.0/lib/rake/application.rb:178:in `standard_exception_handling'
     # /var/lib/jenkins/workspace/elastic+logstash+5.0+multijob-os-compatibility/os/amazon/vendor/bundle/jruby/1.9/gems/rake-12.0.0/lib/rake/application.rb:77:in `run'
     # /var/lib/jenkins/workspace/elastic+logstash+5.0+multijob-os-compatibility/os/amazon/vendor/bundle/jruby/1.9/gems/rake-12.0.0/exe/rake:27:in `(root)'

Finished in 2 minutes 56.4 seconds (files took 3.51 seconds to load)
2210 examples, 1 failure

Failed examples:

rspec /var/lib/jenkins/workspace/elastic+logstash+5.0+multijob-os-compatibility/os/amazon/logstash-core/spec/logstash/agent_spec.rb:488 # LogStash::Agent metrics after config reloading when reloading a bad config sets the failure reload timestamp

Randomized with seed 45405
```

**Comment by jsvd** [comment]
this is a failure in 5.0 branch, are we still backporting bug fixes to < 5.3 branches? if not it may make sense to disable builds for those.

**Comment by colinsurprenant** [comment]
as discuss, if we only push security fixes to < 5.3 I am +1 on disabling builds and only doing manual builds if we need to do a release. @suyograo - is that confirmed for < 5.3 ?

**Comment by jakelandis** [comment]
Closing since 5.0 is quite old by now and there have been many changes around the handling of Timestamp recently.
